pkg/arvo/gen/ivory.hoon

```diff
@@ -38,7 +40,35 @@
   (swat sub (rain /sys/[nam]/hoon .^(@t cx+(welp sys /[nam]/hoon))))
 ::
 ++  build-lib
-  |=  [sub=(trap vase) nam=term]  ^-  (trap vase)
+  |=  [sub=(trap vase) imp=? nam=term]  ^-  (trap vase)
   ~>  %slog.[0 leaf+"ivory: building /lib/{(trip nam)}"]
-  (swat sub (rain /lib/[nam]/hoon .^(@t cx+(welp lib /[nam]/hoon))))
+  =/  hun=hoon
+    %+  mist  /lib/[nam]/hoon
+    .^(@t cx+(welp lib /[nam]/hoon))
+  ?.  imp  (swat sub hun)
+  (swel sub [%ktts nam hun])
```

Comment:
@joemfb idk if this is strictly necessary or not, but using `+swel` for `/lib/vere` gave me a pill that caused the binary to crash.